config/intellij-idea-inspections.xml

```diff
@@ -1997,6 +1997,29 @@
     <inspection_tool class="SuperClassHasFrequentlyUsedInheritors" enabled="true" level="ERROR" enabled_by_default="true" />
     <inspection_tool class="SuperTearDownInFinally" enabled="false" level="ERROR" enabled_by_default="false" />
     <inspection_tool class="SuppressionAnnotation" enabled="true" level="ERROR" enabled_by_default="true">
+        <scope name="Tests" level="ERROR" enabled="true">
```

Comment:
You can use scope for the certain option to make less changes to the config.
See https://github.com/timurt/checkstyle/blob/d8b5b759e859f66c8c5e932bce7eb5906e626912/config/intellij-idea-inspections.xml#L967